cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+        pf.verifyManageStudentPermissionsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled(false);
+        pf.verifyCreateManageProjectPermissionFormsLink();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled();
+
+        cy.log('Click the Create / Manage Project Permission Forms link; then verify that only the Manage Student Permissions link is enabled');
+        pf.clickCreateManageProjectPermissionFormsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled(false);
+
+        cy.log('Click the Create New Permission Form button, fill in values in the dialog; but then click Cancel');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME, 'google.com', true);
+
+        cy.log('Fill in the Create New Permission Form dialog again, but this time, click the Save button');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+
+        cy.log('Create more Permission Forms, to be used to test the Archive, Unarchive & Delete buttons');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+            
+        cy.log("Select a Project from the dropdown (on the 'Create New Permission Form' tab)");
+        pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
+
+        cy.log('Test the links on the Permission Forms table, starting with URL text, Archive, and Unarchive');
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+
+        // TODO: fix clicking on archived rows, then uncomment these:
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'2');
+        // pf.clickPermissionFormArchive(TEST_PERMISSION_FORM_NAME+'3');
+        // pf.clickPermissionFormUnarchive(TEST_PERMISSION_FORM_NAME+'3');
+
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
```

Comment:
@johnTcrawford Since lines 68 to 70 are commented on, we should also comment these lines 72 to 74 until the Archive functionality is uncommented. This looks like we are again validating the same things that we did on lines 62 - 65.

Reply:
You're right that these lines are completely redundant now; but I wanted to leave these lines as they will be once the clicking archived rows issue is resolved.